docs/tutorials/migrations/nvm.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Node Version Manager (nvm)
+description: Using Flox to replace nvm
+---
+
+# Node Version Manager (nvm) migration guide
+
+Flox is an environment and package manager that allows you install software from an extensive catalog into individual [environments][environment_concept]{:target="\_blank"}, each of which usually corresponds to a software project. Flox gives you the opportunity to simplify the development workflow used by your team, consolidating various functions into a single command: `flox activate`.
+
+Just like [nvm](https://github.com/nvm-sh/nvm){:target="\_blank"}, Flox allows you to install and activate different versions of Node.js, allowing developers to switch versions as project requirements dictate.
+
+## Why you might want to use Flox instead of nvm
+nvm does exactly what it purports to do: it manages Node.js versions simply and effectively. Point notwithstanding, it's also one more dependency that you don't have to worry about if you're using Flox. Consider whether one of the following cases applies to you, or to your team:
+
+* **You work with other engineers who are not JavaScript developers.** Flox will allow your team members who are not TypeScript or JavaScript developers to get up and running on your project, with the correct Node.js version. They won't have to install and use an unfamiliar toolchain in a language with which they might not be familiar. Instead, they can use well known Flox commands to activate their dev environments.
+* **You are already using Flox for your non-JS/non-TS projects.** If you are already using Flox for projects written in other languages, you can streamline your development workflow by using it for your TypeScript and JavaScript projects as well. If you can replace a tool that only manages Node.js versions with something that manages _all_ software dependencies, you're well on your way to simplifying your dev workflow.
+* **You need to manage versions of other JavaScript runtimes, like Bun or Deno.** While nvm is, as the name denotes, used for managing Node.js versions, that's all it does. You can use Flox to install node, but you can also use it to install different JavaScript runtimes, including [Bun](https://bun.sh/){:target="\_blank"} and [Deno](https://deno.com/){:target="\_blank"}.
```

Comment:
```suggestion
* **You need to manage versions of other JavaScript runtimes, like Bun or Deno.** While `nvm` is, as the name denotes, used for managing Node.js versions, that's all it does. You can use Flox to install node, but you can also use it to install different JavaScript runtimes, including [Bun](https://bun.sh/){:target="\_blank"} and [Deno](https://deno.com/){:target="\_blank"}.
```

Reply:
As above, I'm referring to nvm the product, so I'm going to keep it formatted as regular text.